public/locales/pt/editor.json

```diff
@@ -0,0 +1,16 @@
+{
+	"editor-desktop-optimized": "Esse site é otimizado para o desktop",
+	"editor-visit-desktop": "Por favor visite readme.so em um desktop para criar o seu readme!",
+	"nav-download": "Download",
+	"download-readme-generated": "Readme Gerado!",
+	"download-reach-out": "Obrigado por usar o readme.so! Sinta-se livre para entrar em contato comigo no",
+	"download-feedback": "com qualquer feedback.",
+	"download-coffee": "Se você achou esse projeto útil, considere me dar um suporte com uma xícara de café!",
+	"editor-column-editor": "Editor",
+	"editor-select": "Selecione uma seção da barra lateral a esquerda para editor os conteúdos",
+	"preview-column-preview": "Preview",
```

Comment:
Do you think there is a more Portuguese word for Download, Editor, Preview and Raw? Asking incase it makes sense to translate those as well or leave them in native english itself

Reply:
`Editor` have the same translation haha
`Download` we use download here for it, the translation for it is `baixar` but we rarely it here
`Preview` we can change it to `Pré visualização`
`Raw` we can change it to `código fonte`